libass/ass_fontselect.c

```diff
@@ -625,7 +625,7 @@ static char *select_font(ASS_FontSelector *priv, ASS_Library *library,
                          ASS_FontStream *stream, uint32_t code)
 {
     ASS_FontProvider *default_provider = priv->default_provider;
-    ASS_FontProviderMetaData meta = {0};
+    ASS_FontProviderMetaData meta = { .families = 0 };
```

Comment:
Relevant GCC bug reports: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=36750 (fixed ages ago), https://gcc.gnu.org/bugzilla/show_bug.cgi?id=53119 (the fix broke, so refixed in 5.0?), https://gcc.gnu.org/bugzilla/show_bug.cgi?id=64709 (the fix had a bug, fixed in 5.0?), https://gcc.gnu.org/bugzilla/show_bug.cgi?id=80454 (open).

Relevant Clang bug report: https://bugs.llvm.org/show_bug.cgi?id=21689 (open)